File: front/src/models/imc.ts
import { Aluno } from "./aluno";

export interface IMC {
  imcId?: string;
  alunoId: string;
  aluno?: Aluno;
  altura: number;
  peso: number;
  valorIMC?: number;
  classificacao?: string;
  criadoEm?: string;
}

File: Prova-Sub/API/API/models/IMC.cs
namespace API.Models;
public class IMC
{
    public string IMCId { get; set; } = Guid.NewGuid().ToString();
    public string AlunoId { get; set; }
    public Aluno? Aluno { get; set; }
    public double Altura { get; set; }
    public double Peso { get; set; }
    public double ValorIMC { get; set; }
    public string Classificacao { get; set; } = string.Empty;
    public DateTime CriadoEm { get; set; } = DateTime.Now;
}
